app/controllers/reminders/new.js

```diff
@@ -7,6 +7,6 @@ export default Ember.Controller.extend({
       model.save().then(()=> {
         this.transitionToRoute('reminders.reminder', model)
       })
-    }
+    },
```

Comment:
we had more than one action saved on new.js but the other was removed. I missed the comma